docs/generator/buildyaml.sh

```diff
@@ -37,7 +37,7 @@ navpart() {
 	if [ -z "$maxdepth" ]; then maxdepth=1; fi
 	if [[ -n $excludefirstlevel ]]; then mindepth=2; else mindepth=1; fi
 
-	for f in $(find $dir -mindepth $mindepth -maxdepth $maxdepth -name "${file}.md" -printf '%h\0%d\0%p\n' | sort -t '\0' -n | awk -F '\0' '{print $3}'); do
+	for f in $(find $dir -mindepth $mindepth -maxdepth $maxdepth -name "${file}.md" -printf '%h|%d|%p\n' | sort -t '|' -n | awk -F '|' '{print $3}'); do
```

Comment:
I _think_ I know why @cosmix added these `\0`(s) -- It has to do with looping constructs; my earlier comment on using the "while read" form should be used here; But I'll approve this change anyway since AFAIK `for x in y; do` in Sh can't operate on inputs separated by null characters.